vm/src/pyobject.rs

```diff
@@ -575,7 +582,7 @@ impl PyContext {
     {
         PyObject::new(
```

Comment:
You could use `self.new_rustfunc_from_box()` here.

Reply:
Yeah, I was also thinking I might add a `PyNativeFuncFactory` impl for the boxed version so that we don't even need two separate functions.